schema/partner.js

```diff
@@ -86,6 +91,24 @@ const PartnerType = new GraphQLObjectType({
           }));
         },
       },
+      artworks: {
+        type: new GraphQLList(Artwork.type),
+        args: {
+          size: {
+            type: GraphQLInt,
+          },
+          sort: ArtworkSorts,
+          exclude: {
+            type: new GraphQLList(GraphQLString),
+          },
+        },
+        resolve: ({ id }, options) => {
+          return gravity(`filter/artworks`, assign({}, options, {
```

Comment:
It's a one line fix in Gravity, just add a `sort Artwork::SORTS` to the params block: https://github.com/artsy/gravity/blob/6a34c53cbf50b20b0f658111c2ea2a7d36432d14/app/api/v1/partners_artworks_endpoint.rb#L65